Monthly Release 2021.07.0 Developer Documentation Updates (#2303)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/release-notes/2021/index.md

Additionally, you can set or update the Logo or Notes fields for any of your Okta app integrations using the API. <!--OKTA-405943-->

#### New Domains API response properties available

The [Domains API](/docs/reference/api/domains) includes the new response object properties of `certificateSourcetype` and `expiration`. The `certificateSourcetype` is a required property that indicates whether the Certificate is provided by the user. The accepted value is `Manual`. The `expiration` property on the DNSrecord object is an optional property that defines the TXT record expiration. <!--OKTA-403600-->

#### Rate limit events for user and session user activity

Previously, rate limit violation events for user and session user were logged as org-wide system events (`system.org.rate_limit.violation`). These rate limit violation events are now logged more specifically as [operation events](/docs/reference/rl-system-log-events/#operation-rate-limits) (`system.operation.rate_limit.violation`). <!--OKTA-394607-->

#### Schema API feature unique attributes is now GA in Preview

> The Schemas API now includes [unique attributes](/docs/reference/api/schemas/#uniqueattributes) for custom properties in Okta user profiles and the Okta Group profile. You can declare a maximum of five unique properties for each user type and five unique properties in the Okta group profile. This feature helps prevent the duplication of data and ensures data integrity. <!--OKTA-400824-->

Reviewer: is it Group or group? User or user?

Author: Appears to be capitalized. Will update

#### Bugs fixed in 2021.07.0

> - The IdP claim was not available in the `id_token` or included with the Token Inline Hook request. (OKTA-407459)

Reviewer: ```suggestion
- The IdP claim wasn't available in the `id_token` or included with the Token Inline Hook request. (OKTA-407459)
```

- When the Users lifecycle API `users/{{userId}}/lifecycle/reset_factors` was called to reset user factors, a status 403 error was received, even with a valid bearer token and scope (`okta.users.manage`). (OKTA-404613)

> - When an OIDC client app was created, the [Apps API](/docs/reference/api/apps) call could not modify the `visibility.hide` property. (OKTA-399408)

Reviewer: ```suggestion
- When an OIDC client app was created, the [Apps API](/docs/reference/api/apps) call couldn't modify the `visibility.hide` property. (OKTA-399408)
```
